man/systemd.environment-generator.xml

```diff
@@ -121,6 +121,90 @@
     </example>
   </refsect1>
 
+  <refsect1>
+    <title>Environment Generator Path</title>
+
+    <para>Environment generators are executed from a set of paths determined
+    during compilation, described in the two tables below. Environment
+    generators found in directories listed earlier override ones with the
+    same name in directories lower in the list.</para>
```

Comment:
This duplicates the text in DESCRIPTION above. It doesn't really matter which, but one of them needs to go.